includes/Core/Util/Tracking.php

```diff
@@ -106,4 +122,67 @@ private function inline_js_base_data( $data ) {
 
 		return $data;
 	}
+
+	/**
+	 * Gets tracking routes.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $routes Array of routes.
+	 * @return array Modified array of routes that contains tracking related routes.
+	 */
+	private function get_rest_routes( $routes ) {
+		$valid_user = function() {
+			return is_user_logged_in();
```

Comment:
This should rather use a Site Kit base capability, e.g. whether the user can authenticate. Not anybody should be able to use this route (e.g. non-administrators right now).

Reply:
Updated.